src/main/io/transponder_ir.c

```diff
@@ -40,7 +40,8 @@
 PG_REGISTER_WITH_RESET_TEMPLATE(transponderConfig_t, transponderConfig, PG_TRANSPONDER_CONFIG, 0);
 
 PG_RESET_TEMPLATE(transponderConfig_t, transponderConfig,
-    .data = { 0x12, 0x34, 0x56, 0x78, 0x9A, 0xBC } // Note, this is NOT a valid transponder code, it's just for testing production hardware
+    .data = { 0x12, 0x34, 0x56, 0x78, 0x9A, 0xBC, 0x0, 0x0, 0x0 }, // Note, this is NOT a valid transponder code, it's just for testing production hardware
+    .provider = ILAP
```

Comment:
at this early stage you can put provider first, and add a .reserved padding byte after it so that .data is word-aligned, likely then we'd only need to bump/change the size of the .data member if additional systems are added.